src/components/colorSwatch/index.tsx

```diff
@@ -148,7 +149,7 @@ class ColorSwatch extends PureComponent<Props> {
         onPress={this.onPress}
         style={[this.styles.container, style]}
         onLayout={this.onLayout}
-        {...this.getAccessibilityInfo()}
+        {...accessibilityInfo}
```

Comment:
Why is the previous approach slow?

Reply:
We added a condition to get the accessibility info only when the user is on accessibility mode since the calculation we do for getting the color name is pretty heavy.